data/phi0472/phi001/phi0472.phi001.perseus-eng3.xml

```diff
@@ -31,18 +31,18 @@ schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-
   <biblStruct>
     <monogr>
       <author>Catullus</author>
-      <title>Carmina</title><idno type="OCLC">878062</idno>
+      <title>The Carmina of Gaius Valerius Catullus.</title>
       <respStmt>
```

Comment:
We don't generally use the `<respStmt>` here anymore. We use `<author> <title> <editor>`
If the editor is not clear, it need not appear.

You could use <author> and <title> with the title being the longer title here. (The short title is fine for the CTS metadata.)

But we definitely don't use the <respStmt> here

Reply:
Hey @lcerrato thanks for all the comments. I think I've made all the fixes but have one last question by long title do you mean the whole title above:
"The Carmina of Gaius Valerius Catullus. Now first completely Englished into verse and prose, the metrical part by Captain Sir Richard F. Burton, and the prose portion, introduction and notes explanatory and illustrative by Leonard C. Smithers." or just "The Carmina of Gaius Valerius Catullus". Thanks!

Reply:
I mean the whole thing, since that captures the two individuals involved.

Reply:
OK I will do that, fix the files and update it one more time.